### PR TITLE
EventGrid - update change log

### PR DESCRIPTION
**Pull request by YijunXieMS**
Update CHANGELOG to release eventgrid

### CODE REVIEW

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-## 2.0.0-beta.4 (Unreleased)
-
+## 2.0.0-beta.4 (2020-02-10)
+### Breaking changes
+- `CloudEvent` constructor now accepts parameter "data". Removed `setData()`.
+- `CloudEvent.parse()` and `EventGridEvent.parse()` are renamed to `fromString()`.
+- `CloudEvent::getData()` of CloudEvent and EventGridEvent now returns `com.azure.core.util.BinaryData`. 
+  Users can use methods `BinaryData` to deserialize event data. The generic version of `getData()` is then removed.
+- Removed `CloudEvent::getDataAsync()`
+- Added `EventGridSasGenerator` class and removed `EventGridSasCredential`. Use `EventGridSasGenerator.generateSas()` to
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:10]: The generateSas method will be moved to the client right? Will that be done in the next release?

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+  create a Shared Access Signature and use `AzureSasCredential` to build a `EventGridPublisherClient`.
+- Renamed `sendEvents` to `sendEventGridEvents`
+
+### Dependency Updates
+- Update `azure-core` dependency to `1.13.0`.
+- Update `azure-core-http-netty` dependency to `1.8.0`.
+- Remove dependency on `azure-core-serializer-json-jackson`.
+- Removed dependency on `commons-io`.
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:18]: This is not necessary as it was only a test dependency.

**Review by srnagar — APPROVED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:10]: Yes, next release.

**Review by hemanttanwar — APPROVED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md]: Removed

**Pull request merged**